### ISSUE TITLE
Cannot be built on Linux

### ISSUE THREAD

**Issue by dougpuob** [dougpuob/cpp-namelint#4]
![image](https://user-images.githubusercontent.com/3453968/52291278-fb600b00-29ac-11e9-9b87-1f1efe0cef2e.png)

```
In file included from /usr/include/c++/8/backward/strstream:50,
                 from /home/dougpuob/working-folder/namelint.git/Source/Config.cpp:4:
/usr/include/c++/8/backward/backward_warning.h:32:2: warning: #warning This file includes at least one deprecated or antiquated header which may be removed without further notice at a future date. Please use a non-deprecated interface with equivalent functionality instead. For a listing of replacement headers and interfaces, consult the file backward_warning.h. To disable this warning use -Wno-deprecated. [-Wcpp]
 #warning \
  ^~~~~~~
/home/dougpuob/working-folder/namelint.git/Source/Config.cpp: In member function ‘bool namelint::Config::LoadStream(std::__cxx11::string)’:
/home/dougpuob/working-folder/namelint.git/Source/Config.cpp:48:40: error: cannot bind non-const lvalue reference of type ‘std::istream&’ {aka ‘std::basic_istream<char>&’} to an rvalue of type ‘std::basic_istream<char>’
     toml::ParseResult pr = toml::parse(istringstream(ConfigContent));
                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /home/dougpuob/working-folder/namelint.git/Source/Config.cpp:9:
/home/dougpuob/working-folder/namelint.git/Source/../Module/tinytoml.git/include/toml/toml.h:378:20: note:   initializing argument 1 of ‘toml::ParseResult toml::parse(std::istream&)’
 inline ParseResult parse(std::istream& is)
                    ^~~~~
CMakeFiles/namelint.dir/build.make:134: recipe for target 'CMakeFiles/namelint.dir/Source/Config.cpp.o' failed
make[2]: *** [CMakeFiles/namelint.dir/Source/Config.cpp.o] Error 1
CMakeFiles/Makefile2:69: recipe for target 'CMakeFiles/namelint.dir/all' failed
make[1]: *** [CMakeFiles/namelint.dir/all] Error 2
Makefile:151: recipe for target 'all' failed
make: *** [all] Error 2
dougpuob@dougpuob-Virtual-Machine:~/working-folder/namelint.git/Build/linux$ 
```
